externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperationSuite.scala

```diff
@@ -575,7 +575,9 @@ class FlinkOperationSuite extends WithFlinkSQLEngine with HiveJDBCTestHelper {
       val metaData = resultSet.getMetaData
       assert(metaData.getColumnType(1) === java.sql.Types.ARRAY)
       assert(resultSet.next())
-      assert(resultSet.getObject(1).toString == "[\"v1\",\"v2\",\"v3\"]")
+      // Adapt to Flink 1.15
+      assert(resultSet.getObject(1).toString == "[\"v1\",\"v2\",\"v3\"]" || resultSet.getObject(
```

Comment:
could you please use `if (flink_1_14) xxx else xxx` here?

Reply:
`"[\"v1\",\"v2\",\"v3\"]"` => `"""["v1","v2","v3"]"""`

Reply:
I will create a new PR to change `if (flink_1_14) xxx else xxx`. This new PR will unify the version until classes.